src/apps/loyalty/containers/inquiries/index.tsx

```diff
@@ -33,7 +33,7 @@ class Inquiries extends React.Component<RelayProps, any> {
       const { __id, artwork } = edge.node
       return (
         <InquiryContainer>
-          <Artwork key={__id} artwork={artwork as any} />
+          <Artwork key={__id} artwork={artwork as any} style={{"max-height": "450px"}} />
```

Comment:
I updated the Artwork component to make it more flexible in the following [PR] (https://github.com/artsy/reaction-force/pull/73/files#diff-c8fcd3ab8d5f69dc01029122cb566ea0L6) as well, so we'll probably have some merge conflicts to resolve 😭 😅. 

But in general, I think using the `style` property to modify base components can be confusing in terms of communicating the proper usage to whoever ends up using it. IMO a better approach would be to create a size enum with different options for the variant versions of the artwork brick we have.